src/gpu/nvidia/sycl_cuda_helper.hpp

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
```

Comment:
Year of copyright should be on 2022. You may also want to put your own copyright below.

Reply:
fixed